app/views/includes/search-form.nunjucks

```diff
@@ -11,8 +11,13 @@
         <h2>{{ searchErrorLabel }}</h2>
         <p>{{ errorMessage }} </p>
       {% elif gps.length === 0 %}
-        <h2>We can't find a surgery matching '{{search}}'</h2>
-        <p>Check the text you entered is right. You might get better results if you enter a full name. Or you can search with a postcode instead.</p>
+        {% if postcode %}
+          <h2>We can't find a surgery near to '{{processedSearch}}'</h2>
```

Comment:
There doesn't seem to be a test of this. 

YO62 7LQ is a remote postcode which returns no results.